src/site/artifacts/cargo_dist.rs

```diff
@@ -292,3 +291,16 @@ pub fn build_list(manifest: &DistManifest, config: &Config) -> Result<Box<div<St
     </div>
     ))
 }
+
+fn create_download_link(config: &Config, name: &String, version: Option<String>) -> String {
+    if let (Some(repo), Some(v)) = (&config.repository, version) {
+        let version_to_use = if v.contains('v') {
+            v.split('v').collect::<Vec<&str>>()[1]
+        } else {
+            v.as_str()
+        };
+        format!("{}/releases/download/v{}/{}", repo, version_to_use, name)
+    } else {
+        String::new()
```

Comment:
if there's no repo or version i feel like we should error here?

Reply:
Done!